subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/impl/FinishedReport.kt

```diff
@@ -0,0 +1,19 @@
+package com.avito.android.test.report.impl
+
+import com.avito.android.test.report.ReportState
+import com.avito.logger.LoggerFactory
+import com.avito.logger.create
+
+internal class FinishedReport(
```

Comment:
finished report can only log `unexpectedFailedTestCase`
We could discuss what else it can do